code/modules/mob/living/simple_animal/hostile/terror_spiders/actions.dm

```diff
@@ -64,7 +137,12 @@
 	return ..()
 
 
-
+/obj/effect/spider/terrorweb/bullet_act(var/obj/item/projectile/Proj)
+	if(Proj.damage_type != BRUTE && Proj.damage_type != BURN)
+		visible_message("<span class='danger'>[src] is undamaged by [Proj]!</span>")
```

Comment:
style wise, it would be probably better to `return` this and just run parent without any else statement.